tests/ImageSharp.Benchmarks/Image/DecodeJpeg.cs

```diff
@@ -23,21 +24,21 @@ public void ReadImages()
         {
             if (this.jpegBytes == null)
             {
-                this.jpegBytes = File.ReadAllBytes("../ImageSharp.Tests/TestImages/Formats/Jpg/Baseline/Calliphora.jpg");
+                this.jpegBytes = File.ReadAllBytes("../../../../../../../../ImageSharp.Tests/TestImages/Formats/Jpg/Baseline/Calliphora.jpg");
```

Comment:
I have plans to fix stuff like this. The test image repository provider code should be shared between tests and benchmarks.